opus/import/obs_cassini_common_pds4.py

```
@@ -8,10 +8,114 @@
+    ##############################################################
+
+    def field_obs_mission_cassini_obs_name(self):
+        return self._some_index_col('cassini:observation_id')
+
+    def field_obs_mission_cassini_spacecraft_clock_count1(self):
+        raw = self._index_col('cassini:spacecraft_clock_start_count')
+        if raw is None:
+            return None
+        try:
+            return opus_support.parse_cassini_sclk(str(raw).strip())
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)